docs/source/package_reference/loading_methods.mdx

```diff
@@ -22,6 +22,12 @@ Methods for listing and loading datasets and metrics:
 
 ## Metrics
 
+<Tip warning={true}>
+
+Metrics will soon be deprecated in 🤗 Datasets. To learn more about how to use metrics, take a look at our newest library 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index)! In addition to metrics, we've also added more tools for evaluating models and datasets.
```

Comment:
(nit) maybe replace "our" by "the" ? Just in case we don't want to confuse users who don't know who's "us" in a context of a warning message x)

the same applies for all the other messages

```suggestion
Metrics will soon be deprecated in 🤗 Datasets. To learn more about how to use metrics, take a look at the newest library 🤗 [Evaluate](https://huggingface.co/docs/evaluate/index)! In addition to metrics, we've also added more tools for evaluating models and datasets.
```

Reply:
Done.